packages/website/src/components/Benefits/Benefits.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+
+import { BenefitCard, BenefitCardProps } from '@coderscamp/ui/components/BenefitCard';
+import { Button } from '@coderscamp/ui/components/Button';
+import { Center } from '@coderscamp/ui/components/Center';
+import { Grid } from '@coderscamp/ui/components/Grid';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import { SolidArrowDownIcon, SolidArrowUpIcon } from '@coderscamp/ui/icons';
+
+export interface BenefitsProps {
+  title: string;
+  benefitItems: BenefitCardProps[];
+  showExpandProfitButton?: boolean;
+}
+
+const expandedProfitButton = {
+  props: {
+    position: 'sticky',
+    bottom: '20px',
+    icon: <SolidArrowUpIcon />,
+  },
+  text: 'Zwiń listę zalet CodersCamp',
+} as const;
+const rolledUpProfitButton = {
+  props: {
+    icon: <SolidArrowDownIcon />,
+  },
+  text: 'Rozwiń listę zalet CodersCamp',
+} as const;
+
+const BENEFITS_AFTER_SHRINKAGE = 6;
+
+export const Benefits = ({ title, benefitItems, showExpandProfitButton = false }: BenefitsProps) => {
+  const columnsCount = useBreakpointValue({ base: 1, md: 2, xl: 3 } as const);
+  const profitButtonSize = useBreakpointValue({ base: 'sm', md: 'lg' } as const);
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const profitButtonValues = isExpanded ? expandedProfitButton : rolledUpProfitButton;
+  const handleProfitList = () => {
```

Comment:
💭 `handleExpandButtonClick` could be a better name. Also, I think we should address the page jump after the user shrinks the list
![CleanShot 2021-08-22 at 15 18 40](https://user-images.githubusercontent.com/22938071/130356787-abfaa85f-c05f-462a-a4f5-20c5937b2319.gif)